docs/body.md

```diff
@@ -1819,6 +1819,7 @@ Changes
 
 -   Fix Microsoft Windows fails to model 2008 Server Cluster Disks (ZPS-2015)
 -   Fix Windows - traceback when no data returned for wmi class win32_computersystem (ZPS-2384)
+-   Fix Cluster Disk state always 'Inherited' (ZPS-2416)
 
```

Comment:
Why do we have both a Mediawiki and Markdown? Shouldnt there be only one source?

Reply:
b/c windows is special.  it's a free zenpack, so i keep the wiki page up to date.  also, the wiki is where we are linking to download the zenpack.  so, i might as well keep it up for now